content/actions/learn-github-actions/migrating-from-travis-ci-to-github-actions.md

```diff
@@ -39,6 +39,24 @@ Travis CI lets you set environment variables and share them between stages. Simi
 
 Travis CI and {% data variables.product.prodname_actions %} both include default environment variables that you can use in your YAML files. For {% data variables.product.prodname_actions %}, you can see these listed in "[Default environment variables](/actions/reference/environment-variables#default-environment-variables)."
```

Comment:
Huy

Reply:
igrating-from-travis-ci-to-github-actions.md
@@ -39,6 +39,24 @@ Travis CI lets you set environment variables and share them between stages. Simi

 Travis CI and {% data variables.product.prodname_actions %} both include default environment variables that you can use in your YAML files. For {% data variables.product.prodname_actions %}, you can see these listed in "[Default environment variables](/actions/reference/environment-variables#default-environment-variables)."

 To help you get started, this table maps some of the co

Reply:
🤚